engine/src/main/java/org/terasology/rendering/opengl/fbms/DisplayResolutionDependentFBOs.java

```diff
@@ -131,5 +132,6 @@ public void swapReadWriteBuffers() {
         WRITE_ONLY_GBUFFER.setFbo(fbo);
         fboLookup.put(READ_ONLY_GBUFFER.getName(), READ_ONLY_GBUFFER.getFbo());
         fboLookup.put(WRITE_ONLY_GBUFFER.getName(), WRITE_ONLY_GBUFFER.getFbo());
+        notifySubscribers();
```

Comment:
Looking at this line I noticed that in update() subscribers are notified in createFBOs and _before_ updateDefaultFBOs() is called. If I am correct you might want to move the notifySubscribers() line from createFBOs to the end of the IF block in update(), just after the line updateDefaultFBOs().